posthog/models/cohort.py

```diff
@@ -71,6 +71,7 @@ def create(self, *args: Any, **kwargs: Any):
 
 class Cohort(models.Model):
     name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    description: models.CharField = models.CharField(max_length=400, blank=True)
```

Comment:
Same constraint on description size as name size sounds too restrictive?

Reply:
Well, 400 seems like probably too long for a cohort name and should be enough for most descriptions, but alright, increased the limit here to 1000

Reply:
That makes sense too! Don't mind going the reverse way, assuming no one has that long cohort names 😋